### ISSUE TITLE
Add more information to the Reddit comment reply.

### ISSUE THREAD

**Issue by nmur** [nmur/reddit-video-rotation-bot#45]
Information such as: 
- Link to source code
- PM link to /u/nmur
- How to use
- Look at other bots for other things

**Comment by nmur** [comment]
```
Rotated Video: https://giant.gfycat.com/WellmadeElaborateFirebelliedtoad.mp4  
  
***  
^^[source](https://github.com/nmur/reddit-video-rotation-bot)&nbsp;-&nbsp;[pm&nbsp;me](https://www.reddit.com/message/compose/?to=nmur)
```

**Comment by nmur** [comment]
Addressed by #49
